src/main/java/com/google/sps/data/Recommended.java

```diff
@@ -0,0 +1,68 @@
+package com.google.sps.data;
+
+import java.util.*;
+
+public class Recommended {
+
+  public static Drink getDrink(String storeid, String drinkName) {
+    List<Drink> drinklist = DrinkDAO.getDrinksByStore(storeid);
+    for (Drink d : drinklist) {
+      if (d.getName().equals(drinkName)) {
+        return d;
+      }
+    }
+    return null;
+  }
+  // given a coffeeshop, return its tydrecommended score
+  public static double getScore(Drink d) {
+    double comments = d.getNumRatings();
+    // instead of shop rating get drink rating from this store
+    double rating = d.getRating();
+    double distanceMinutes = getDistanceMins();
+    double commentScore, ratingScore, distanceScore;
+    //      /30             /35         /35
+
+    // get commentScore
+    if (comments >= 100.0) // 100+
+    commentScore = 30;
+    else if (comments >= 75.0) // 75-99
```

Comment:
Use braces to be consistent with the rest of the code.

Reply:
will do